test/cpp/microbenchmarks/bm_chttp2_transport.cc

```diff
@@ -426,7 +426,7 @@ static void BM_TransportStreamSend(benchmark::State& state) {
     grpc_slice_buffer_add(&send_buffer, grpc_slice_ref(send_slice));
     send_stream.Init(&send_buffer, 0);
     grpc_slice_buffer_destroy(&send_buffer);
-    // force outgoing window to be yuge
```

Comment:
This is not a typo.
https://knowyourmeme.com/memes/yuge

Reply:
We are in the bay, not in new york :D